raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -120,7 +118,11 @@ def close(self, partner_address: Address) -> None:
 
         if rtc_partner is not None:
             aiortc_channel_close_callback(rtc_partner)
-            rtc_partner.peer_connection.close()
+            self._spawn_web_rtc_coroutine(
+                coroutine=rtc_partner.peer_connection.close(),
```

Comment:
you most likely don't want to "spawn" the result of `close()` but just the function `close`, no?

Reply:
the function `_spawn_web_rtc_coroutine()` is wrapping a coroutine into a greenlet. 

This is preferred to be used for every coroutine because it stores the greenlet in a set which is going to be joined when stop_even is set. To reduce the unexpected behavior, I try to always call a coroutine with that wrapping function in order to gracefully shut the transport down.

Reply:
Inside it calls the `aiogevent` function `yield_future` which awaits on a future in gevent land

Reply:
I see, thanks for the clarification!